src/main/java/StringCalculator/StringCalculator.java

```diff
@@ -0,0 +1,51 @@
+package StringCalculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class StringCalculator {
+    public int stringSum(Object addValues) {
+        if (validateNullOrEmpty(addValues))
+            return 0;
+        List<Integer> splitAddValues = stringSplitAndParaInt(addValues);
+        validateNegative(splitAddValues);
+        return splitAddValues.stream().reduce(Integer::sum).get();
+    }
+
+    private boolean validateNullOrEmpty(Object addValues) {
+        if (addValues == null || addValues == "") {
+            return true;
+        }
+        return false;
```

Comment:
이부분은 이렇게도 쓸 수 있을거 같아요. 
```suggestion
addValues == null || addValues == "";
```

Reply:
감사합니다. 이렇게 간결하게 쓸 수 있었군요. 반영하여 수정하겠습니다.